terraform/ec2/asg/amazon-cloudwatch-agent.json

```diff
@@ -5,16 +5,12 @@
   },
   "traces": {
     "traces_collected": {
-      "app_signals": {
-        "enabled": true
-      }
+      "application_signals": {}
```

Comment:
Just confirming, we don't see `enabled` here anymore?

Reply:
Yes